packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -314,6 +315,11 @@ export class PerfEntryRecorder {
   }
 
   stop() {
+    /**
+     * Empties out the performance entries from the
+     * current observer
+     */
+    this.po.takeRecords()
```

Comment:
Just a defensive mechanism to clear out the entries if we don't have any listeners taking care of the records.

Reply:
Not required, tested it with different scenarios and we handle all the records properly. Also safari does not have the `takeRecords` which needs extra unnecessary `typeof === function` check. Hence removed it.